src/routes/quotation/+page.server.ts

```diff
@@ -127,7 +134,7 @@ export const actions = {
 } satisfies Actions;
 
 export const load = (async () => {
-  // Get the last quotation made
+  // Replace this with QuotationRepository.findById(id: number)
```

Comment:
really this should not exist at all because of the race condition... replace with redirect to /quotation/id